src/main/java/net/sf/jabref/sql/importer/DbImportAction.java

```diff
@@ -196,4 +197,29 @@ public void update() {
         frame.output(Localization.lang("Imported %0 databases successfully", Integer.toString(databases.size())));
     }
 
+    /**
+     * Run using Spin features to put each method on the correct thread.
+     *
+     * @throws Throwable
```

Comment:
This can be removed as it does not provide any additional information.

OK for me to move such a util class to here. As The DbImportAction will be gone soon (see #1451), this dependency on spin will go, too. :tada: